lib/core/types.hpp

```diff
@@ -19,23 +21,19 @@ struct CanFrame {
   std::array<std::uint8_t, 8> data;
 };
 
-// current trajectory struct
 struct Trajectory {
   Float displacement;
   Float velocity;
   Float acceleration;
 };
 
-// number of each type of sensors
 static constexpr std::uint8_t kNumImus     = 4;
 static constexpr std::uint8_t kNumAxis     = 3;
 static constexpr std::uint8_t kNumEncoders = 4;
 static constexpr std::uint8_t kNumKeyence  = 2;
 
-// data format for raw sensor data
-using RawImuData  = std::array<std::array<Float, kNumAxis>, kNumImus>;
-using ImuData     = std::array<Float, kNumImus>;
-using EncoderData = std::array<std::uint32_t, kNumEncoders>;
-using KeyenceData = std::array<std::uint32_t, kNumKeyence>;
+using RawImuData     = std::array<std::array<Float, kNumAxis>, kNumImus>;
```

Comment:
Think these are changing soon to have timestamps too. Similar idea to last years datapoint.

Reply:
That makes sense; even more of a reason to split them between core and navigation.